src/coreclr/dlls/mscorrc/mscorrc.rc

```diff
@@ -724,28 +724,28 @@ END
 // Profiler messages for event log
 STRINGTABLE DISCARDABLE
 BEGIN
-        IDS_E_PROF_BAD_PATH                     "Loading profiler failed.  COR_ENABLE_PROFILING and COR_PROFILER were set properly, but COR_PROFILER_PATH was not.  COR_PROFILER_PATH must be set to the full path of the profiler DLL to load with no more than 260 charaters including the null terminator.\n"
-        IDS_E_PROF_NO_CLSID                     "Loading profiler failed.  COR_ENABLE_PROFILING was set properly, but COR_PROFILER was not.  COR_PROFILER must be set to the CLSID of the profiler to load.\n"
-        IDS_E_PROF_INTERNAL_INIT                "Loading profiler failed due to an internal profiling services initialization failure.  Profiler CLSID: '%s'.  HRESULT: 0x%x.\n"
-        IDS_E_PROF_BAD_CLSID                    "Loading profiler failed.  COR_PROFILER is set to an invalid CLSID: '%s'.  HRESULT: 0x%x.\n"
+        IDS_E_PROF_BAD_PATH                     "Loading profiler failed.  COR_ENABLE_PROFILING and COR_PROFILER were set properly, but COR_PROFILER_PATH was not.  COR_PROFILER_PATH must be set to the full path of the profiler DLL to load with no more than 260 characters including the null terminator.\n"
```

Comment:
These env variables were renamed in .NET Core to `CORECLR_ENABLE_PROFILING` to avoid collisions with .NET Framework.

Reply:
All the updates can be found at https://docs.microsoft.com/dotnet/core/runtime-config/debugging-profiling